src/utils/mapPostprocessor.js

```diff
@@ -108,6 +108,12 @@ function upgrade(map) {
 		map['_burstSliders'] = [];
 	}
 
+	[...map._notes].forEach(note => {
```

Comment:
Maybe move this to a separate function and check NE requirements before calling it?